track/track_test.go

```diff
@@ -0,0 +1,76 @@
+package track
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestTrack(t *testing.T) {
+
+	t.Run("Track 1 data", func(t *testing.T) {
+		tracker := NewTrackFirst(true)
+
+		samples := []string{
+			`B4815881002867896^YATES/EUGENE JOHN         ^21129821000123456789`,
+			`B4815881002861896^YATES/EUGENE L            ^^^356858      00998000000`,
+			`B4000340000000506^John/Doe                  ^13011110000123000`,
+			`B1234567890123445^PADILLA/L.                ^99011200000000000000**XXX******`,
+		}
+		for _, sample := range samples {
```

Comment:
It would be great if we test parsed value here as well. From my experience, it's not enough to read/write without error. We should check what exactly we get in `Number`, `Expiration` and other fields.